prog/vm/vm_host_slice_nexus.rb

```diff
@@ -0,0 +1,132 @@
+# frozen_string_literal: true
+
+class Prog::Vm::VmHostSliceNexus < Prog::Base
+  subject_is :vm_host_slice
+
+  semaphore :destroy, :start_after_host_reboot, :checkup
```

Comment:
Since the subject is `VmHostSlice`, and the model has semaphores. No need to define semaphores again at prog.

https://github.com/ubicloud/ubicloud/pull/2417